zooniverse/front-end-monorepo
# lib-classifier: Refactor survey task ConfusedWith styling

## Package
- lib-classifier

## Linked Issue and/or Talk Post
- towards #4557 (item 2)

## Describe your changes
- refactor DropButton and Confusion styling

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
  - local: https://local.zooniverse.org:8080/?project=1634&workflow=2434
- What user actions should my reviewer step through to review this PR?
  - click a confusion when the button is towards the top of screen
  - click a confusion when the button is towards the bottom of screen
- Which storybook stories should be reviewed?
  - choice story - http://localhost:6006/?path=/story/tasks-survey-choice--default
  - full survey story - http://localhost:6006/?path=/story/tasks-survey--default
- Are there plans for follow up PR’s to further fix this bug or develop this feature? no

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Bug Fix
